src/main/java/Main.java

```diff
@@ -59,6 +59,7 @@ public static boolean is_valid_password(String password) throws IllegalArgumentE
 
     //account has a schedule instance that is worked on
     public static void populate_allcourses() throws IOException {
+        allcourses = new ArrayList<>();
```

Comment:
What is the decision behind calling allcourses here and not in load_accounts?

Reply:
It makes sense to initialize allcourses when we're going to fill it rather than in some other place.... I wanted to be able to use populate_allcourses in tests and with '`allcourses = new ArrayList<>();`' in load_accounts you get an exception